grpc-all/1.68.0.wso2v1/META-INF/services/io.grpc.ManagedChannelProvider

```diff
@@ -0,0 +1 @@
+io.grpc.netty.shaded.io.grpc.netty.NettyChannelProvider
```

Comment:
missing license header

Reply:
These are some class files. We cannot add license for them